onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -2115,6 +2118,35 @@ TEST_F(GraphTransformationTests, BiasGeluTest) {
   ASSERT_TRUE(op_to_count["com.microsoft.BiasGelu"] == 1);
 }
 
+#ifdef ENABLE_TRAINING
```

Comment:
you can place this under orttraining/test/optimizer/graph_transformer_test.cc

The file is for training only graph transformers

Reply:
Done.